api/controllers/Cid_EntryController.js

```diff
@@ -71,6 +71,12 @@ module.exports = {
       .populate('female_active_movements_in')
       .populate('female_active_movements_out')
       .then(centres => _.map(centres, centre => Centres.publishUpdate(centre.id, centre)))
+      .then(() => {
```

Comment:
you don't need to return the movements, but you should return the centres.update promise so you can write:
`then(() => Centres.update({})`